Assets/Scripts/Game/PlayerDeath.cs

```diff
@@ -162,8 +160,66 @@ private void PlayerEntity_OnDeath(DaggerfallEntity entity)
             currentCameraHeight = startCameraHeight;
             DaggerfallUI.Instance.FadeHUDToBlack(FadeDuration);
 
+            // There are 3 pain-like sounds for each race/gender. The third one, used here, sounds like
+            // it may have been meant for when the player dies.
+            SoundClips sound = SoundClips.None;
+            PlayerEntity playerEntity = GameManager.Instance.PlayerEntity;
+            switch (playerEntity.RaceTemplate.ID)
+            {
+                case (int)Races.Breton:
+                    if (playerEntity.Gender == Genders.Male)
+                        sound = SoundClips.BretonMalePain3;
+                else
+                        sound = SoundClips.BretonFemalePain3;
```

Comment:
Would be more compact written in a method call like this:

```c#
sound = GetRaceGenderPain3Sound((Races) playerEntity.RaceTemplate.ID)
```

```c#
SoundClip GetRaceGenderPain3Sound(Races race)
{
    switch ()
    {
        case Races.Breton:
            return (playerEntity.Gender == Genders.Male) ? SoundClips.BretonMalePain3 : SoundClips.BretonFemalePain3;
        case Races.Redguard:
             return (playerEntity.Gender == Genders.Male) ? SoundClips.RedguardMalePain3 : SoundClips.RedguardFemalePain3;
        // .... etc ....
    }
}
```

Just a suggestion, functionally it's fine.

Reply:
Sure, I'll rewrite that way.